components/shared/view/BasicImage.vue

```diff
@@ -1,32 +1,52 @@
 <template>
-  <b-image
-    :src="src || placeholder"
-    :src-fallback="placeholder"
-    :alt="alt"
-    ratio="1by1"
-    :class="customClass"
-    :rounded="rounded"
-    data-cy="type-image"
-    @error="onImageError">
-    <template #placeholder>
-      <NeoSkeleton full-size no-margin :circle="rounded" />
-    </template>
-  </b-image>
+  <figure class="image-wrapper image is-1by1" :class="customClass">
```

Comment:
@roiLeo let's do the custom ration via CSS if it's just like that

Reply:
❓ I want to keep default 1by1 and not introduce some breaking change
With Buefy ratio is calculated to add a `padding-top: %;` , let's say I have `ratio=16by9` then we will have something like `padding-top: 56.25%;`